src/main/java/CommandSet/DueCommand.java

```diff
@@ -0,0 +1,52 @@
+package CommandSet;
+
+import Helper.TaskList;
+import Helper.Ui;
+
+/**
+ * File contains the implementation of DueCommand class.
+ * <h1>DueCommand</h1>
+ * <p>
+ * DueCommand implements the command that the would returns the list of tasks
+ * that are due on a certain date or due before a certain date.
+ * </p>
+ *
+ * @author Saravanan Anuja Harish
+ */
+public class DueCommand extends Command {
+
+    // stores the command for due before.
+    private static final String DUE_BEFORE = "due-before";
+
+    // stores the command for due on.
+    private static final String DUE_ON = "due-on";
+
+    /**
+     * Constructor for DueCommand.
+     * returns an instance of DueCommand.
+     */
+    DueCommand() {
+        super();
+    }
+
+    /**
+     * checks the tasks that are due on date.
+     * @param date the due date.
+     * @param taskList the list of user tasks.
+     */
+    public static void tasksDueOn(String date, TaskList taskList) {
+        date = date.substring(DUE_ON.length()).trim();
```

Comment:
Should the name be a verb? like getTasksDueOn

Reply:
Yup, corrected it to be a verb, now looks better! 👍